src/apps/compound/common/compound.borrow.contract-position-fetcher.ts

```diff
@@ -117,6 +123,13 @@ export abstract class CompoundBorrowContractPositionFetcher<
   ): Promise<CompoundBorrowTokenDataProps> {
     const [underlyingToken] = params.contractPosition.tokens;
 
+    const multicall = params.multicall;
+    const erc20 = this.appToolkit.globalContracts.erc20({
+      address: params.contractPosition.address,
+      network: this.network,
+    });
+    const symbol = await multicall.wrap(erc20).symbol();
```

Comment:
Supply and borrow position share the same addresses meaning symbol will **always** be available. Symbol was needed in order to show something like:
`88 zWETH = 1 WETH`